src/libs/Network/SequentialQueue.ts

```diff
@@ -271,5 +273,15 @@ function waitForIdle(): Promise<unknown> {
     return isReadyPromise;
 }
 
-export {flush, getCurrentRequest, isRunning, isPaused, push, waitForIdle, pause, unpause, process};
+function resetQueue(): void {
```

Comment:
NAB: Please also explain that this is only for tests

Reply:
fixed